rero_ils/modules/patron_transactions/extensions.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Patron transactions record extensions."""
+
+from flask_babelex import gettext as _
+from invenio_records.extensions import RecordExtension
+
+from ..patron_transaction_events.api import PatronTransactionEvent
```

Comment:
We use absolute import if class belongs to another module :: Should be "rero_ils.modules.patron_transaction_events.api" (same for `utils` below)